Dockerfile

```diff
@@ -233,6 +238,10 @@ RUN cd /docker-py \
 	&& pip install paramiko==2.4.2 \
 	&& pip install yamllint==1.5.0 \
 	&& pip install -r test-requirements.txt
+COPY --from=rootlesskit /build/ /usr/local/bin/
+# VPNKit git b4c8b69e68f74c69a6e2fff696a3a196b061dde6 (1/5/2019)
+# FIXME: currently, this always install amd64 binary
```

Comment:
Wonder if we can build from source; there's a Dockerfile in the repo to build vpnkit https://github.com/moby/vpnkit/blob/master/Dockerfile, but not sure we should copy that (perhaps the steps from the Dockerfile could be move into the `Makefile`? @djs55 - think that would work?

Reply:
I think cross-compilation is actively being worked on in OCaml, see https://discuss.ocaml.org/t/ocaml-cross-compiler/1494 . I don't think it works 100% yet :(

I think the only reliable way to build `vpnkit` for other targets would be to build on those targets. The current `vpnkit` CI builds binaries for macOS and Windows (for use in Docker Desktop). Which other targets do you need?

Reply:
Linux for armhf, arm64, s390x, and ppc64le

Reply:
@AkihiroSuda Btw, we don't have s390x and ppc64le releases for `18.09` anymore.

Reply:
@tonistiigi @thaJeztah does that mean that we can actually consider to drop s390x and ppc64le CI builds from Moby?

Reply:
I haven't been part of this discussion, but I don't see why moby would need to drop them from CI as moby is a source project to Docker 18.xx and future 19.xx products; whether Docker has official releases seems unrelated to whether those architectures are still validated in the moby upstream project during CI.

Reply:
Yes, no need to drop CI but if it becomes troublesome to support them for new features it is a data point to consider. For example, I think if we can't have these new binaries available on these platforms it shouldn't block this PR.